cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/approximation-accuracy.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Text from 'antd/lib/typography/Text';
+import Slider from 'antd/lib/slider';
+
+interface Props {
+    approxPolyAccuracy: number;
+    onChange(value: number): void;
+}
+
+export function thresholdFromAccuracy(approxPolyAccuracy: number): number {
+    const maxAccuracy = 7;
+    const approxPolyMaxDistance = maxAccuracy - approxPolyAccuracy;
+    let threshold = 0;
+    if (approxPolyMaxDistance > 0) {
+        // 0.5 for 1, 1 for 2, 2 for 3, 4 for 4, ...
+        threshold = 2 ** (approxPolyMaxDistance - 2);
```

Comment:
Probably I don't understand but the minimum value is 0.25 now. Right? Is it expected? Need to fix the comment or code.

```suggestion
        threshold = 2 ** (approxPolyMaxDistance - 1);
```

Reply:
Comment is not precise a bit, minimum value was intended is 0.25 for approxPolyMaxDistance 0.

Reply:
2 ** (1 - 2) = 0.5
2 ** (2 - 2) = 1
2 ** (3 - 2) = 2
2 ** (4 - 2) = 4

A separated condition if approxPolyMaxDistance === 0, the threshold is also 0 (no approximation)